media-sound/bitwig-studio/bitwig-studio-1.3.16.ebuild

```diff
@@ -0,0 +1,82 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+# TODO
+# - What about bundled libs? Shouldn't they be removed?
+# - Install docs in correct location? (currently in /opt/bitwig-studio/resources/doc)
+
+EAPI=6
+inherit eutils fdo-mime unpacker
+
+DESCRIPTION="for creation and performance of your musical ideas on stage or in the studio."
```

Comment:
We should probably change this to something like "Multi-platform music-creation system for production, performance and DJing" (from their website)